system-tests/services/_scripts/setup-external-volumes

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# create simple app
+cat <<EOF | dcos marathon app add
```

Comment:
Can we add this output to results? Feels like dcos fails to start the app and then the tests fails because there's no app, but no way to debug this.